specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/quotas.json

```diff
@@ -0,0 +1,271 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01",
+    "title": "ContainerRegistryManagementClient",
+    "description": "Admin Container Registry Management Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/quotas/{quotaName}": {
+      "put": {
+        "x-ms-examples": {
+          "Create or update an existing container registry quota.": {
+            "$ref": "./examples/Quotas/CreateOrUpdate.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryQuotas"
+        ],
+        "description": "Create or update an existing container registry quota.",
+        "operationId": "ContainerRegistryQuotas_CreateOrUpdate",
```

Comment:
same comment on ContainerRegistry prefix

Reply:
It is ok, we don't want to collide with Storage Api for example which also has similar operations.

Reply:
I made the change but it generates the wrong ps cmdlet name : Set-Asz.ps1 what is not appropriates. I kept it as is for now.